docs/content/installation/integrations/app-protect-waf-v5/configuration.md

```diff
@@ -0,0 +1,195 @@
+
+NGINX Ingress Controller supports OpenAPI Specification versions 2.0 and 3.0.
+
+The simplest way to create an API protection policy is using an OpenAPI Specification file to import the details of the APIs. If you use an OpenAPI Specification file, NGINX App Protect WAF will automatically create a policy for the following properties (depending on what's included in the spec file):
+
+- Methods
+- URLs
+- Parameters
+- JSON profiles
+
+An OpenAPI-ready policy template is provided with the NGINX App Protect WAF packages and is located in: `/etc/app_protect/conf/NginxApiSecurityPolicy.json`
+
+It contains violations related to OpenAPI set to blocking (enforced).
+
+## Configuration in NGINX Plus Ingress Controller using Virtual Server Resource
+
+In this example we deploy NGINX Ingress Controller with NGINX Plus and NGINX App Protect WAF v5, deploy a simple web application, and then configure load balancing and WAF protection for that application using the VirtualServer resource.
+
+{{< note >}} You can find the example, and the files referenced, on [GitHub](https://github.com/nginxinc/kubernetes-ingress/tree/v3.6.0/examples/custom-resources/app-protect-waf/app-protect-waf-v55).{{< /note >}}
+
+## Prerequisites
+
+1. Follow the installation [instructions]({{< relref "installation/integrations/app-protect-waf-v5/installation.md" >}}) to deploy NGINX Ingress Controller with NGINX Plus and NGINX App Protect WAF.
+
+2. Save the public IP address of NGINX Ingress Controller into a shell variable:
+
+   ```shell
+    IC_IP=XXX.YYY.ZZZ.III
+   ```
+
+3. Save the HTTP port of NGINX Ingress Controller into a shell variable:
+
+   ```shell
+    IC_HTTP_PORT=<port number>
+   ```
+
+### Step 1. Deploy a Web Application
+
+Create the application deployment and service:
+
+  ```shell
+  kubectl apply -f https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v3.6.0/examples/custom-resources/app-protect-waf-v5/webapp.yaml
+  ```
+
+### Step 2. Deploy the Policy Bundle
+
+Create the syslog service and pod for the NGINX App Protect WAF security logs:
+
+   ```shell
+   kubectl apply -f https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v3.6.0/examples/custom-resources/app-protect-waf-v5/syslog.yaml
+   ```
+
+### Step 3 - Deploy the WAF Policy
+
+Create the WAF policy
+
+ ```shell
+  kubectl apply -f https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v3.6.0/examples/custom-resources/app-protect-waf-v5/waf.yaml
+ ```
+
+  Note the NGINX App Protect WAF configuration settings in the Policy resource. They enable WAF protection by configuring NGINX App Protect WAF with the policy and log configuration created in the previous step.
```

Comment:
This should call out the need to create a bundle as a prereq

Reply:
addressed